Gemfile.lock

```diff
@@ -207,7 +206,7 @@ GEM
     nokogiri (1.12.5)
       mini_portile2 (~> 2.6.1)
       racc (~> 1.4)
-    nokogiri (1.12.5-x86_64-darwin)
+    nokogiri (1.12.5-arm64-darwin)
```

Comment:
```suggestion
    nokogiri (1.12.5-x86_64-darwin)
```

Reply:
Argh, so annoying!  I have updated my version of Ruby to no effect. 😞

Reply:
I wonder if there's is some way of pining it so it stays correct....